scripts/contentScript.js

```diff
@@ -139,15 +139,82 @@ async function getHighlightedPR(repoOwner, repoName, userId) {
 };
 
 // for showing all tracked/ untrack pr in a organization
-async function getTrackedPR(orgName) {
+async function getRepoInfo(orgName) {
 	const body = { "org": `${orgName}` }
 	const url = 'https://gcscruncsql-k7jns52mtq-el.a.run.app/setup/repos';
 	const trackedRepos = await apiCall(url, body);
-	if(trackedRepos){
-		cssForGithubTrackRepo(trackedRepos);
+	if (trackedRepos) {
+		return trackedRepos;
 	};
 }
 
+async function getTrackedRepo(orgName) {
+	const data = await getRepoInfo(orgName);
+	if (data) cssForGithubTrackRepo(data);
+}
+
+
+// adding favButton
+async function addButton(orgName, orgRepo) {
+
+	const addedRepoList = await getRepoInfo(orgName);
+
+	if (!addedRepoList.includes(orgRepo)) {
+		const PrSection = document.getElementById('repo-content-pjax-container');
+		// for vibinex logo
+		const img = document.createElement("img");
+		img.setAttribute('id', 'vibinexWarningLogo')
+		img.src = "https://vibinex.com/favicon.ico";
+		img.style.width = '35px';
+		img.style.height = '35px';
+		img.style.borderRadius = '35px';
+		img.style.position = 'fixed';
+		img.style.left = '30px';
+		img.style.bottom = '50px';
+		img.style.cursor = 'pointer'
+		// for redirecting if the repo is not added 
+		const redirectLink = document.createElement('a');
+		redirectLink.href = 'https://www.vibinex.com';
+		redirectLink.style.position = 'fixed';
+		redirectLink.style.left = '50px';
+		redirectLink.style.bottom = '45px';
+		redirectLink.style.zIndex = '101';
+		// for adding plusIcon
+		const plusIcon = document.createElement('img');
+		plusIcon.src = "https://img.freepik.com/free-icon/add-button-with-plus-symbol-black-circle_318-48599.jpg";
+		plusIcon.style.width = '35px';
+		plusIcon.style.height = '35px';
+		plusIcon.style.borderRadius = '35px';
+		plusIcon.style.cursor = 'pointer';
+		redirectLink.appendChild(plusIcon);
+
+		const infoBanner = document.createElement('div');
+		infoBanner.setAttribute('id', 'vibinex-info');
+		// tooltip value on hover 
```

Comment:
why did you have to do all this? Could you just add the "title" attribute to the inserted HTML?

Reply:
why adding into title ? this div is for toolTip

Reply:
added title instead of id

Reply:
instead of "id"? Title and id are very different things. You identify elements using `id`. You create a tooltip by using `title`. It is not the best-designed tooltip, but it works.

Reply:
But you can leave it be. This is alright too.

Reply:
all changes made,